FirebaseAdmin/FirebaseAdmin.IntegrationTests/FirebaseAuthTest.cs

```diff
@@ -510,6 +510,115 @@ public async Task ListUsers()
             }
         }
 
+        [Fact]
+        public async Task ImportUsersBasic()
+        {
+            var randomUser = RandomUser.Create();
+            var args = new ImportUserRecordArgs()
+            {
+                Uid = randomUser.Uid,
+                Email = randomUser.Email,
+                DisplayName = "Random User",
+                PhotoUrl = "https://example.com/photo.png",
+                EmailVerified = true,
+            };
+
+            var options = new UserImportOptions();
+            IEnumerable<ImportUserRecordArgs> usersLst = new List<ImportUserRecordArgs>();
+            usersLst = usersLst.Append(args);
+
+            var resp = await FirebaseAuth.DefaultInstance.ImportUsersAsync(usersLst);
+
+            try
+            {
+                Assert.Equal(1, resp.SuccessCount);
```

Comment:
Also call `GetUserAsync()` on the UID to ensure the user account has been created.